server/models/comment.js

```diff
@@ -85,9 +21,10 @@ const CommentSchema = new Schema({
   post_id: {
```

Comment:
forgot this one :p

Reply:
I was hoping you'd be OK with it.  I've lost a lot of time on other projects having mismatches between postId and postID.

Reply:
ahaha, ok that's fair